eZ/Publish/Core/Persistence/Legacy/Content/Type/Mapper.php

```diff
@@ -235,6 +235,7 @@ public function extractFieldFromRow(array $row, array $multilingualData = [])
         $field->fieldType = $row['ezcontentclass_attribute_data_type_string'];
         $field->isTranslatable = ($row['ezcontentclass_attribute_can_translate'] == 1);
         $field->isRequired = $row['ezcontentclass_attribute_is_required'] == 1;
+        $field->isThumbnail = $row['ezcontentclass_attribute_is_thumbnail'] == 1;
```

Comment:
Following previous comments:
```suggestion
        $field->isThumbnail = !empty($row['ezcontentclass_attribute_is_thumbnail'])
```
because the result depends on db driver (unless it actually returns true/false for BOOL/TINYINT(1) for Maria, MySQL, and Postgres - didn't check).